### DISCUSSION TITLE
Error while opening b2nd array in windows

### DISCUSSION THREAD

**Discussion by leoyala** [Blosc/python-blosc2#291] [Q&A]
Hello, 

I created an ND-Array in Ubuntu and shared it for other people to use: https://www.synapse.org/Synapse:syn52432292. 
Loading the file in ubuntu is not a problem but in windows I am getting the following error when doing `a = blosc2.open(r'G:\subject_01.b2nd', mode='r')`

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\...\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\blosc2\schunk.py", line 997, in open
    return blosc2_ext.open(urlpath, mode, offset, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "blosc2_ext.pyx", line 1699, in blosc2.blosc2_ext.open
RuntimeError: blosc2_schunk_open_offset('G:\\subject_01.b2nd', 0) returned NULL
```

I already tested the consistency of the file and it seems to be correct. I would appreciate your help with this 🙂 

Details: 
blosc2  2.7.1
Python 3.12
Windows 11 23H2 64-Bit

**Comment by FrancescAlted** [comment]
Hmm, the double backslash in urlpath seems suspicious.  We have not a Windows platform to test, but if you can fix the issue and want to contribute a PR, you are welcome!
